test/e2e/certificates_test.go

```diff
@@ -435,6 +452,202 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("dns-01 challenge using explicit credentials on ibm cloud CIS", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
+			cisCRN, isCisCRN := os.LookupEnv(cisCRNEnvironmentVar)
+			if targetPlatform, ok := os.LookupEnv(targetPlatformEnvironmentVar); ok && targetPlatform == "ibmcloud" {
+				if !isCisCRN || cisCRN == "" {
+					Fail("cisCRN is required for IBM Cloud platform")
+				}
+			} else {
+				Skip("skipping as the cluster does not use IBM Cloud CIS")
+			}
+
+			By("creating a test namespace")
+			ns, err := loader.CreateTestingNS("e2e-acme-explicit-dns01")
+			Expect(err).NotTo(HaveOccurred())
+			defer loader.DeleteTestingNS(ns.Name)
+
+			By("creating new certificate ClusterIssuer with IBM Cloud CIS webhook solver")
+			clusterIssuerName := "letsencrypt-dns01-explicit"
+			By("delete existing ClusterIssuer with the same name if it exists")
+			_ = certmanagerClient.CertmanagerV1().ClusterIssuers().Delete(ctx, clusterIssuerName, metav1.DeleteOptions{})
```

Comment:
Not sure if this `Delete` operation will affect other cases (with same `clusterIssuerName`) that are running in parallel.

Reply:
It's a great question. We are following the style of the prior tests, and wrapping them where we are using a webhook to test on IBM ppc64le and s390x. Are there recommendations on updating? should we? we could create with a random hash postfix.

Reply:
I'd say add your own random suffix, and use the same suffix at the the time of deletion; that'd make most sense.